src/App.tsx

```diff
@@ -1,14 +1,68 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
+import { Filters } from './types/Filters';
+
+function getVisibleTodos(
+  allToDos: Todo[],
+  {
+    query,
+    filter,
+  }: {
+    query: string;
+    filter: Filters;
+  },
+) {
+  let handledTodos = [...allToDos];
```

Comment:
```suggestion
  let handledTodos = allToDos;
```
Filter returns new array

Reply:
Done!